### DISCUSSION TITLE
.AppImage version's windows glitching since 0.8.2 came out (Linux Mint 21.3)

### DISCUSSION THREAD

**Discussion by LavaPlaysOfficial** [modrinth/.github#284] [Other discussions]
**### _I COULDN'T FIND AN ARTICLE ON THE OFFICIAL SITE_**

Since the new **0.8.2** update came out my windows is **glitching** and is a head ache to work with, i couldnt find a article for this on the official Modrinth site so i came here.

Ive tried the **.deb** package, it wont even open so i went back to the **.AppImage** version.

The windows c**onstanly glitches** and sometimes the whole **windows goes blank.**

I have 2 monitors a _1360x768_ and a _1920x1080_ one, it glitches on **both**, it didnt do this till 0.8.2.

**Comment by Minenash** [comment]
That doesn't sound good, hopefully it gets fixed soon (I'm not a modrinth dev). Unfortunately the discussion threads isn't for bug reports, those go [here](https://github.com/modrinth/code/issues). I would transfer it, but there's a template for bug reports there
